include/swift/AST/Expr.h

```diff
@@ -2300,7 +2300,7 @@ class SubscriptExpr final : public LookupExpr,
     return Bits.SubscriptExpr.HasArgLabelLocs;
   }
 
-  /// Whether this call with written with a trailing closure.
+  /// Whether this call was written with a trailing closure.
```

Comment:
This should be a separate patch - if the other fix gets reverted for whatever reason your typo fix will live on :)

Reply:
Submitted as https://github.com/apple/swift/pull/17098.